rules/utils/get-docs-url.js

```diff
@@ -0,0 +1,13 @@
+'use strict';
+
+const repoUrl = 'https://github.com/sindresorhus/eslint-plugin-unicorn';
+
+function getDocsUrl(ruleName, givenCommitHash) {
```

Comment:
Export this one directly.

Reply:
Rename `givenCommitMash` to `commitHash`. Below, write it like this

```js
commitHash = commitHash || 'master';
```

Reply:
> Export this one directly.

The rest of this was written in CommonJS, so that's the style I was following. I'll switch it to a `module` 👍.

> Rename `givenCommitHash` to `commitHash`.

It's generally a bad idea to assign to parameters, but I can change it to that style.

Reply:
I meant `module.exports = (ruleName, commit:ash) => {`

Reply:
Strings are immutable. It's just the same as a default value, only one line extra. It's a bad idea when an object is passed in and you'd write `obj.commitHash = obj.commitHash || 'master'`